docs/modules/generic_send.md

```diff
@@ -0,0 +1,77 @@
+# Generic Send (generic_send.py)
+
+## Description
+
+This script is used to send data generated by the [PerunDataGenerator](PerunDataGenerator.md) to the Perun API. It is
+used by the service-specific sending scripts written in python.
```

Comment:
it can be called by send script written in any language, now most commonly by bash scripts. the service-specific wording can be misplaced with destination type service-specific, maybe it would be clearer to use some other word. it's just the send method that can be called directly by other python script.